pkg/server/plugin/nodeattestor/aws/iid_test.go

```diff
@@ -234,7 +163,8 @@ func (s *IIDAttestorSuite) TestClientAndIDReturns() {
 		mockExpect                      func(mock *mock_aws.MockClient)
 		expectID                        string
 		expectSelectors                 []*common.Selector
-		expectErr                       string
+		expectErrCode                   codes.Code
+		expectErrMessage                string
```

Comment:
nit: we've named this differently in most other places
```suggestion
		expectCode                   codes.Code
		expectMessage                string
```